vpr/src/draw/draw.cpp

```diff
@@ -990,6 +1019,41 @@ static void drawplace(ezgl::renderer* g) {
                 //Determine the block color
                 ezgl::color block_color;
                 t_logical_block_type_ptr logical_block_type = nullptr;
+#    ifdef VTR_ENABLE_DEBUG_LOGGING
```

Comment:
Needs a comment on what this does and should be moved into a subroutine.

Reply:
I'm not sure what you mean by moving it in a subroutine. VTR_ENABLE_DEBUG_LOGGING is a compile-time flag that is enabled allows debugging